addons/sale_coupon/tests/test_sale_invoicing.py

```diff
@@ -50,3 +50,94 @@ def test_invoicing_order_with_promotions(self):
         self.assertEqual(order.order_line, invoiceable_lines)
         account_move = order._create_invoices()
         self.assertEqual(len(account_move.invoice_line_ids), 2)
+
+    def test_double_invoice_order_with_order_percentage_discount(self):
```

Comment:
```suggestion
    def test_02_double_invoice_order_with_order_percentage_discount(self):
```
and add `01` on the previous one, let's save this file while it is not too late

Reply:
So it should be 'test_01_invoicing_order_with_promotions',
'test_02_double_invoice_order_with_order_percentage_discount' and
'test_02_double_invoice_order_with_free_product' ?

Reply:
I just saw the other comment below, oops

Reply:
No, rename the (only) existing one 01, then add 02 and 03 to your 2 new methods

So
test_01_invoicing_order_with_promotions
test_02_double_invoice_order_with_order_percentage_discount
test_03_double_invoice_order_with_free_product